godtools/App/Features/UserLessonProgress/Presentation/ResumeLessonProgressModal/ResumeLessonProgressModal.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  ResumeLessonProgressModal.swift
+//  godtools
+//
+//  Created by Rachael Skeath on 10/29/24.
+//  Copyright © 2024 Cru. All rights reserved.
+//
+
+import SwiftUI
+
+struct ResumeLessonProgressModal: View {
+    
+    private let buttonHeight: CGFloat = 48
+    private let modalInset: CGFloat = 28
+    private let buttonInset: CGFloat = 20
+    private let buttonSpace: CGFloat = 12
+
+    @ObservedObject private var viewModel: ResumeLessonProgressModalViewModel
+    
+    init(viewModel: ResumeLessonProgressModalViewModel) {
+        self.viewModel = viewModel
+    }
+    
+    var body: some View {
+        GeometryReader { geometry in
+            let totalSpaceAroundModal = modalInset * 2
+            let modalWidth = geometry.size.width - totalSpaceAroundModal
+            let totalSpaceAroundButtons = (buttonInset * 2) + buttonSpace
+            let buttonWidth = (modalWidth - totalSpaceAroundButtons) / 2
+            
+            ZStack {
+                if #available(iOS 15.0, *) {
+                    Color.clear
+                        .edgesIgnoringSafeArea(.all)
+                        .background(.ultraThinMaterial)
+                } else {
+                    VisualEffectView(effect: UIBlurEffect(style: .systemUltraThinMaterial))
+                        .edgesIgnoringSafeArea(.all)
+                }
+                
+                VStack(spacing: 0) {
+                    Text(viewModel.interfaceStringsDomainModel.title)
+                        .font(FontLibrary.sfProTextRegular.font(size: 28))
+                        .foregroundColor(ColorPalette.gtGrey.color)
+                        .padding(.top, 30)
+                        .padding(.bottom, 15)
+                    
+                    Text(viewModel.interfaceStringsDomainModel.subtitle)
+                        .font(FontLibrary.sfProTextRegular.font(size: 16))
+                        .foregroundColor(ColorPalette.gtGrey.color)
+                        .multilineTextAlignment(.center)
+                        .padding(.horizontal, buttonInset + 20)
+                        .padding(.bottom, 35)
+                    
+                    HStack(spacing: buttonSpace) {
+                        GTWhiteButton(title: viewModel.interfaceStringsDomainModel.startOverButtonText, width: buttonWidth, height: buttonHeight) {
```

Comment:
Hey @rachaelblue also on the GTWhiteButton and GTBlueButton can you inject a fontSize here of at least 15.

It defaults to 12 after inspecting the initializer which looks pretty small here.  

Looking at the Figma and I see size 15 is being used.  Sometimes I will even go a point or 2 larger than what Figma suggests if the screen size is small in Figma just depends.